src/audit/pull_from_queue.py

```diff
@@ -81,11 +81,27 @@ async def pull_from_queue_loop():
     """
     logger.info("Starting to pull from queue...")
     aws_sqs_config = config["QUEUE_CONFIG"]["aws_sqs_config"]
+    # we know the cred is in AWS_CREDENTIALS (see `AuditServiceConfig.validate`)
+    aws_creds = (
+        config["AWS_CREDENTIALS"][aws_sqs_config["aws_cred"]]
+        if "aws_cred" in aws_sqs_config
```

Comment:
shouldn't this be `if "aws_cred" in aws_sqs_config and aws_sqs_config["aws_cred"]`? I mean what if we have an empty value for `aws_sqs_config["aws_cred"]` like in -- [config-default#L27](https://github.com/uc-cdis/audit-service/blob/299d0c7ec49778af7d193ed5968641bf86a5bcf6/src/audit/config-default.yaml#L27-L27)

Feel free to correct me if I'm wrong!

Reply:
The default value is `None`, not `{}`

If someone set it to `{}`, it'd be fine, and if they set it to another type's null value such as `[]`, the config validation would error on line 103, which I think is acceptable behavior (erroring when the wrong type is used - although we could add a type check)

Reply:
If the value of `aws_sqs_config["aws_cred"]` is `None` then `config["AWS_CREDENTIALS"][None]` would throw an exception right? Is that what is expected?

Reply:
right we could add a check